plugin.py

```diff
@@ -17,6 +21,25 @@ class LspTailwindcssPlugin(NpmClientHandler):
         server_directory, 'extension', 'dist', 'server', 'index.js'
     )
 
+    @classmethod
+    def is_allowed_to_start(
+        cls,
+        _window: sublime.Window,
+        _initiating_view: Optional[sublime.View] = None,
+        workspace_folders: Optional[List[WorkspaceFolder]] = None,
+        _configuration: Optional[ClientConfig] = None
+    ) -> Optional[str]:
+        if not workspace_folders:
+            return "{} - requires a folder to start.".format(cls.package_name)
+        path = workspace_folders[0].path
+        tailwind_config_file_path = os.path.join(path, 'tailwind.config.js')
```

Comment:
You want to maybe check what happens when there are multiple workspace folders and some have config and some don't?

Reply:
Glad you asked that question.
This solution is copied from lsp-flow 
I thought that there is a problem with this approach (checking only the first workspace folder), but I concluded that everything works as it should.

I don't know 100% why it works, but here is an explanation:

Lets say that we have 2 workspace folders `FolderA` and `FolderB`.
Lets say FolderA has everything needed for LSP-tailwindcss to start,
while FolderB don't.

If we open ST and have both workspace folders open.
if we open a file in FolderA,
workspace_folders array will be['FolderA', 'FolderB'] (note in WorkspaceFolder in reality is an object, contains path and name keys. I used the string array representation of folders just to easier explain my conclusions)
Note that the FolderA is at index `0`.

If we now open a file in FolderB,
workspace_folders array will be['FolderB', 'FolderA'].
Note that this time FolderB is at index `0`!

The main conclusion, from my testing is that, workspace_folders[0] is always the current workspace folder for the opened file. I am not sure what code in the LSP  repo is responsible for this behavior. thus that is why I wrote that I am not 100% sure why this works.

Reply:
> I am not sure what code in the LSP repo is responsible for this behavior.

It's the `sorted_workspace_folders` function that does that :)

Reply:
Right, that sounds fine then.

Note that it works because the server doesn't support workspaces folders. If it would then there would be one instance of the server started that would handle all folders.